definitions/ext-rfscanner/dashboard.json

```diff
@@ -0,0 +1,167 @@
+{
```

Comment:
This is the new dashboards format. We are not yet supporting this one. So we will either manually change it to the old one or block the PR until we have support for it.

@IreneMLC what do you think is better to do here?

Reply:
We discussed it, we will not add the dashboard in the first iteration